scripts/workspace_test_evaluate.py

```diff
@@ -0,0 +1,298 @@
+"""
+Copyright 2018 IBM Corporation
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import argparse
+import datetime
+import json
+import logging
+import os
+import re
+import sys
+import time
+
+import lxml.etree as LET
+
+from wawCommons import getScriptLogger, openFile, setLoggerConfig
+
+logger = getScriptLogger(__file__)
+
+try:
+    basestring            # Python 2
+except NameError:
+    basestring = (str, )  # Python 3
```

Comment:
Maybe we could fix this now (better than later) - we only support Python 3 from the last release. We can just use `isinstance(s, str)` instead of `isinstance(s, basestring)`